tests/services/test_service_communities.py

```diff
@@ -165,6 +172,74 @@ def test_community_owner_add_record_to_communities(
         assert results.to_dict()["hits"]["total"] == expected_n_results
 
 
+def test_community_owner_add_record_to_communities_forcing_review_with_comment(
+    curator,
+    inviter,
+    community,
+    open_review_community,
+    record_community,
+    service,
+    rdm_record_service,
+    requests_service,
+    events_service,
+    anyuser_identity,
+    community_records_service,
+):
+    """Test owner addition of record to open review by forcing a review with a comment."""
+    test_comment = "Could someone review it?"
+    data = {
+        "communities": [
+            {
+                "id": open_review_community.id,
+                "require_review": True,
+                "comment": test_comment,
+            },
+        ]
+    }
+    inviter(curator.id, open_review_community.id, "curator")
+    record = record_community.create_record(uploader=curator)
+
+    results = service.add(curator.identity, record.pid.pid_value, data)
+    assert not results["errors"]
+    assert len(results["success"]) == 1
+
+    record_item = rdm_record_service.read(curator.identity, record.pid.pid_value)
+    record_communities = record_item.data["parent"]["communities"]["ids"]
+    # assert that the community was not added
+    assert len(record_communities) == 1
+    assert community.id in record_communities
+    assert open_review_community.id not in record_communities
+
+    # assert that the request of the curator is in review ("submitted")
+    for result in results["success"]:
+        community_id = result["community"]
+        request_id = result["request"]
+        request = requests_service.read(curator.identity, request_id).to_dict()
+        assert request["type"] == CommunityInclusion.type_id
+        if community_id == open_review_community.id:
+            assert request["status"] == "submitted"
+            assert request["is_open"] is True
+        else:
+            raise
+        # TODO FIXME: Small delay to ensure that comment is indexed in ES
+        time.sleep(3)
```

Comment:
You can use `refresh`, like [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/tests/fixtures/test_cli.py#L43) or [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/tests/fixtures/test_fixtures.py#L218).